yoyodyne/special.py

```diff
@@ -6,5 +6,6 @@
 START = "<S>"
 END = "<E>"
 UNK = "<UNK>"
+UNK_FEAT = "<UNK_FEAT>"
```

Comment:
I am not sure this really belongs here if it ought not to be part of SPECIAL below. Alternatively it could live solely inside of `datasets`.